scipy/stats/tests/test_discrete_distns.py

```diff
@@ -96,3 +98,15 @@ def test_betabinom_bernoulli():
     p = betabinom(1, a, b).pmf(k)
     expected = bernoulli(a / (a + b)).pmf(k)
     assert_almost_equal(p, expected)
+
+
+def test_zipfian():
```

Comment:
should this test be marked slow? I'm assuming it will be.

Reply:
In the version of the code you commented on, `test_zipfian` is equivalent to the test now called `test_zipfian_asymptotic`. 

For `a` greater than 1, the generalized harmonic numbers are not calculated by taking the sum of a series; they use the zeta function, so the magnitude of `n` doesn't really affect the speed. The test is for `k = np.arange(1, 21)`, and it doesn't take too long to do 21 of these calculations.

This test is slow:
```
7.94s call     stats/tests/test_discrete_basic.py::test_discrete_basic[zipfian-arg17-True]
```
so I should check skip whatever is taking a long time there - maybe `rvs`.

Reply:
these are comments on outdated diff, marking resolved

Reply:
For now I've effectively marked the `zipfian` `test_discrete_basic` as slow, so it only runs when test mode is full.